# Parameter settings for Ghost lane detector node

lane_detector:

# Misc. Parameters
  debug: false          # Output debugging log messages
  bev_ppm: 200          # Pixels-per-meter for Bird's Eye View image
  use_BEV_image: false  # If the BEV image should be used for edge detection and scanning
  
# Scanner Parameters
  min_range: 0.2           # Minimum detactable range for scanner [m]
  max_range: 3.0           # Maximum detactable range for scanner [m]
  scan_res: 100            # Resolution of scan within camera FOV
  scan_frame_id: "scanner" # Name of the frame to attach to the scan message
